backend/frontend/templatetags/custom_tags.py

```diff
@@ -10,7 +10,11 @@ def user_avatar(user, size="default"):
         "default": "w-[30px] h-[30px]",
         "small": "w-[15px] h-[15px] text-xs",
     }
-    initial = user.email[0].upper()
+    if hasattr(user, "email"):
+        initial = user.email[0].upper()
+    else:
+        initial =  user[0].upper()
+        
```

Comment:
Do we still need to do this? We can access the user instances to pass on the template just like we talked about on Slack or how it currently works in develop